template/src/auth/AuthProvider.tsx

```diff
@@ -310,6 +311,16 @@ const AuthProvider = (props: AuthProviderProps) => {
     // Ignore if on sdk since IDP flow is not supported
     // For unauthenticated flow authLogin should be called to get the token
     logger.log(LogSource.Internals, 'AUTH', 'App loaded');
+    if (isRecordingBot) {
+      logger.debug(
```

Comment:
Just to clarify: 
if it's a recording bot, then you don't need to authenticate, instead you directly set authenticated to true.
correct @SupriyaAdep  ?

currently for the authenticatd flow, recording bot is trying to login and create a new user ?

Reply:
In the current implementation we dont set authenticated to true..we override the token if its a bot.  So we let the original code of authentication to run and then override the token. 
I made a mistake by incorrectly understanding the auth flow-> As authwrapper is a parent of all routes so it sets a token and then inside child routes when we encounter a bot route it resets the token again.